SingularityUI/app/handlebarsHelpers.coffee

```diff
@@ -76,21 +76,13 @@ Handlebars.registerHelper 'humanizeText', (text) ->
     text
 
 # 2121 => '2 KB'
-Handlebars.registerHelper 'humanizeFileSize', (fileSize) ->
-    kilo = 1024
-    mega = 1024 * 1024
-    giga = 1024 * 1024 * 1024
-
-    shorten = (which) -> Math.round fileSize / which
-
-    if fileSize > giga
-        return "#{ shorten giga } GB"
-    else if fileSize > mega
-        return "#{ shorten mega } MB"
-    else if fileSize > kilo
-        return "#{ shorten kilo } KB"
-    else
-        return "#{ fileSize } B"
+Handlebars.registerHelper 'humanizeFileSize', (bytes) ->
+    k = 1024
+    sizes = ['B', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB']
+
+    return '0 B' if bytes is 0
+    i = Math.floor(Math.log(bytes) / Math.log(k))
```

Comment:
I know we'd never go beyond YB, but just to be 100% robust we should do `Math.min(Math.floor(Math.log(bytes) / Math.log(k)), sizes.length-1)` to ensure we dont go past the length of sizes